src/Explorer/Notebook/NotebookManager.tsx

```diff
@@ -170,7 +180,17 @@ export default class NotebookManager {
         undefined,
         "Cosmos DB cannot access your Github account anymore. Please connect to GitHub again.",
         "Connect to GitHub",
-        () => this.params.container.openGitHubReposPanel("Connect to GitHub"),
+        () =>
+          useSidePanel
+            .getState()
+            .openSidePanel(
+              "Connect to GitHub",
+              <GitHubReposPanel
+                explorer={this.params.container}
+                gitHubClientProp={this.params.container.notebookManager.gitHubClient}
+                junoClientProp={undefined}
```

Comment:
Same comment as above. Unlcear why we are passing undefined here

Reply:
done